tfx/examples/chicago_taxi_pipeline/taxi_pipeline_with_inference_e2e_test.py

```diff
@@ -0,0 +1,110 @@
+# Lint as: python2, python3
```

Comment:
Just curious, why is this extra E2E test needed?

Reply:
oops, i clearly didn't read all of your initial comment 😓

Reply:
Not sure how keen they will be to add another full test, maybe their [taxi_pipeline_with_inference_test.py](https://github.com/tensorflow/tfx/blob/master/tfx/examples/chicago_taxi_pipeline/taxi_pipeline_with_inference_test.py) could be repurposed. Worth pointing out to them the resulting PR.

Reply:
I'm going to try selling it, although why do you think they might object?

Reply:
A full-blown new test in their code base for a small pattern change could be seen as overkill, I wouldn't think so if it covers more, they'll be the judge of that though.